src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/Blocks/product_preferences_stock.html.twig

```diff
@@ -100,6 +100,16 @@
             {{ form_widget(stockForm.pack_stock_management) }}
           </div>
         </div>
+        <div class="form-group row">
+          {{ ps.label_with_help(
+            ('Display out-of-stock label on product listing pages'|trans),
```

Comment:
no domain for the translation ?

Reply:
Thank you @sowbiba ! I hadn't notice. I assumed it was in Admin.Shopparameters.Feature but you are right it is not written. 

![image](https://user-images.githubusercontent.com/70583503/100445054-c30f4c80-30ac-11eb-8c19-27986517f425.png)

Reply:
@sowbiba @Julievrz you can see the default domain here https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/Blocks/product_preferences_stock.html.twig#L26

Reply:
Thanks!